PrintNodePrintJobOptions.cs

```diff
@@ -73,7 +73,7 @@ public class PrintNodePrintJobOptions
         /// See <a href="https://www.printnode.com/docs/api/curl/#parameters">Parameters</a>.
         /// </summary>
         [JsonProperty("pages")]
-        public string Pages { get; set; }
+        public string Pages { get; set; } = "-";

```

Comment:
Have you tested this? I haven't looked very well, but can't find anything about setting Pages to "-" in the official documentation?

Reply:
Yes, I have tested it but only with a single page.

It is documented in Print Job options under Pages  "- prints all pages."
https://www.printnode.com/docs/api/curl#printjob-creating